proposals/dark_mode_extension_icons.md

```diff
@@ -49,18 +49,25 @@ The Chromium bug has a significant amount of developer interest.
 
 manifest.json
 ```
-"icon_variants": [{
-  "16": "16.png",
-  "32": "32.png",
-}, {
-  "16": "dark16.png",
-  "32": "dark32.png",
-  "color_scheme": "dark",
-}, {
-  "16": "light16.png",
-  "32": "light32.png",
-  "color_scheme": "light"
-}]
+"icon_variants": [
+  {
+    "any": "any.svg",
+  },
```

Comment:
Maybe include an example of `"color_scheme": ["light", "dark"]` here, since you have the example below that has no `color_scheme` too, then all cases are covered.

Reply:
Done.